pkg/littlefs/fs/littlefs_fs.c

```diff
@@ -0,0 +1,499 @@
+/*
+ * Copyright (C) 2017 OTA keys S.A.
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     fs
```

Comment:
should be `pkg_littlefs`?  Btw. that group is not defined, as far as I can see, right? should add a `doc.txt` to do so.

Reply:
True, but actually C files are not parsed by Doxygen.
There is already a `doc.txt` in the pkg directory defining the groups.